src/controllers/despesasController.js

```diff
@@ -0,0 +1,34 @@
+const Despesa = require('../model/Despesas.js');
+
+async function cadastrarDespesa(despesa) {
+  return await Despesa.create(despesa);
+}
+async function listarTodos() {
+  try {
+    return await Despesa.find();
+    
+  } catch (error) {
+    return error
+  }
+}
```

Comment:
Aqui vc está apenas retornando o erro, exigindo que, quem chamar sua função, precisará verificar se o retorno é um erro OU uma lista. Isso não está errado, mas acredito que não seja isso que queira aqui...

Olhando para a linha 26 deste mesmo arquivo, vejo que vc não verifica este retorno. Ao invés disso, vc captura o erro através do _try/catch_. Para que isso funcione, vc **precisa** alterar o `return` para `throw`. Mas, visto que vc não está fazendo nada no _try_ e nem no _catch_ (além de chamar o _find_ e repassar o erro), vc poderia simplesmente retornar o `find`, igual na função `cadastrarDespesa`.

```suggestion
async function listarTodos() {
  return await Despesa.find();
}
```